testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarIntegration.tests.js

```diff
@@ -172,6 +173,9 @@ export default function() {
                 .find(`.${DROPDOWNMENU_CLASS}`)
                 .trigger('dxclick');
 
+            $(`.${DROPDOWNEDITOR_CLASS} .${INPUT_CLASS}`)
```

Comment:
Why do we need to add it here?

Reply:
I've reconsidered the decicion, you may look it again